scripts/main/upload.js

```diff
@@ -198,9 +228,7 @@ upload.start = {
 					$(".basicModal .rows").scrollTop(scrollPos);
 
 					// Set status to processing
-					$(".basicModal .rows .row:nth-child(" + (file_num + 1) + ") .status").html(lychee.locale["UPLOAD_PROCESSING"]);
-					processing_count++;
-					currently_uploading = false;
+					$(nRowStatusSelector(file_num + 1)).html(lychee.locale["UPLOAD_PROCESSING"]);
```

Comment:
Merge problem? You skipped two important lines of code that I added yesterday...

Reply:
Yes indeed - sorry! The merge conflict caught me off guard... it was easy to resolve but I must have inadvertently deleted those two lines as well. Added them back in.